imagetest/test_suites/gveperf/startupscripts/netserver_startup.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# This script installs iperf on a VM and starts an iperf server for the client
+# to test the network bandwidth between the two VMs.
+
+sleepduration=120
+
+if [[ -f /usr/bin/apt ]]; then
+  echo "apt found Installing iperf."
+  sudo apt update && sudo apt install iperf
+elif [[ -f /bin/apt-get ]]; then
+  echo "apt found Installing iperf."
+  sudo apt-get update && sudo apt-get install iperf
+elif [[ -f /bin/dnf ]]; then
+  echo "dnf found Installing iperf."
+  os=$(cat /etc/redhat-release)
+  if [[ "$os" == *"release 9."* ]]; then
+    sudo dnf -y install https://dl.fedoraproject.org/pub/epel/9/Everything/x86_64/Packages/i/iperf-2.1.6-2.el9.x86_64.rpm
```

Comment:
These package calls will likely break over time as versions move. Instead, for EPEL it is better to install the epel-release package and then install iperf from the repo. https://docs.fedoraproject.org/en-US/epel/#how_can_i_use_these_extra_packages